### ISSUE TITLE
LongPoll support

### ISSUE THREAD

**Issue by bigardone** [saschatimme/elm-phoenix#29]
Hi, there!
Are you planning to add support for LongPoll, just like in https://github.com/phoenixframework/phoenix/blob/master/assets/js/phoenix.js#L730 ?
Cheers!

**Comment by saschatimme** [comment]
I don't think that I will personally implement this, but I would be open for a PR.
Regarding a PR: I think it should be a pure Elm implementation and not just a thin wrapper around the phoenix.js implementation.
